scripts/lint-branch.sh

```diff
@@ -18,7 +18,7 @@ then
 else
     echo $LINES
     if [ -n "$FAIL_WARNINGS" ]; then
-        NODE_OPTIONS="--max-old-space-size=8192" eslint --max-warnings=0 $LINES
+        NODE_OPTIONS="--max-old-space-size=8192" eslint --max-warnings=0 $LINES --no-ignore
```

Comment:
whats the reason for adding --no-ignore here?

Reply:
ESLint generates warnings when files are technically ignored, but are being linted anyway (i.e. you are running `eslint [filename] on the file)`. @kurtisassad recommended simply not ignoring files, since the flag we actually want to use (which suppresses warnings) doesn't actually work. 

It's not optimal, but it's a good stopgap.